autogpt_platform/frontend/src/app/reset_password/actions.ts

```diff
@@ -0,0 +1,60 @@
+"use server";
+import getServerSupabase from "@/lib/supabase/getServerSupabase";
+import { redirect } from "next/navigation";
+import * as Sentry from "@sentry/nextjs";
+import { headers } from "next/headers";
+
+export async function sendResetEmail(email: string) {
+  return await Sentry.withServerActionInstrumentation(
+    "sendResetEmail",
+    {},
+    async () => {
+      const supabase = getServerSupabase();
+      const headersList = headers();
+      const host = headersList.get("host");
+      const protocol =
+        process.env.NODE_ENV === "development" ? "http" : "https";
```

Comment:
Don't we have a constant value for this ?

Reply:
We do on the backend, I don't see anything on the frontend but let me know if you have any idea.